docs/UserGuide.md

```diff
@@ -406,6 +371,34 @@ If unindicated, the `School Residence` field will default to `DOES NOT LIVE ON C
 * No appointment should clash with any other appointments.
 * The student that the appointment is for must exist in the records.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## FAQ
+
+**Q**: What do I do if I accidentally delete a student? 
+<br>
+**A**: Unfortunately, the deletion is permanent. You will have to manually re-add the student.
+
+**Q**: What can be included in medical details?
+<br>
+**A**: Allergies and past medical history.
+
+**Q** : Will I be able to add other types of appointments besides vaccination appointments?
+<br>
+**A** : No, the current version only allows you to add vaccination appointments and not any other type of appointments. 
+
+**Q** : Am I able to reassign an appointment to another student?
+<br>
+**A** : No, you will need to add a new appointment for the new student. 
+
+**Q**: Why are vaccinated students able to receive new appointments?
+<br>
+**A**: Beyond the two shots currently required for the approved Pfizer-BioNTech and Moderna vaccines in Singapore, booster shots could be required in the future to prolong immunity and provide protection against fast-emerging COVID-19 variants. 
+
+**Q**: How long is a vaccination appointment?
+<br>
+**A**: The vaccination appointment lasts 30 mins, which includes registration, administering the vaccine and observation. 
```

Comment:
Should this be "A vaccination appt" or "Each..." or "Every" instead of "The"?

Reply:
Yup, each is better. Thanks!